recipes/whatlies/meta.yaml

```diff
@@ -0,0 +1,116 @@
+{% set name = "whatlies" %}
+{% set version = "0.6.5" %}
+
+
+package:
+  name: {{ name|lower }}-meta
+  version: {{ version }}
+
+source:
+  url: https://pypi.io/packages/source/{{ name[0] }}/{{ name }}/whatlies-{{ version }}.tar.gz
+  sha256: 0b97c2b5b0bbc5c4044278ea8b2d3222cbb125e1fbf7bd854bb7f09b557e13d7
+
+build:
+  number: 0
+  noarch: python
+requirements:
+# ref: https://gitter.im/conda-forge/conda-forge.github.io?at=6195afbdb5ba9e5a11c02791
+  host:
+    - python >=3.6,<3.8
+  run:
+    - python >=3.6,<3.8
+
+outputs:
+  - name: {{ name|lower }}
+    build:
+      noarch: python
+      script: {{ PYTHON }} -m pip install . -vv
+      run_exports:
+        - {{ pin_subpackage(name, min_pin='x.x', max_pin='x.x') }}
```

Comment:
```suggestion
```
Are you sure you'll need this? It's normally used by compiled packages.

Reply:
Honestly, no, am not sure. It was in one of the recipes I looked into. My "guess" was that `run_exports + pin_subpackage` is creating the base-package, that gets used downstream. However, contrary to this *assumption* of mine, I also found plenty of recipes, that do not use `run_exports + pin_subpackage`; and yet they do get by just fine! So, I was beginning to question, if I should drop it. Thank you.